src/styles/components/side-tabs.less

```diff
@@ -1,76 +1,100 @@
 .sidebar-tabs {
 
-  display: flex;
-  flex-direction: column;
-  list-style-type: none;
-  margin: 0;
-  margin-top: @base-spacing-unit * 0.5;
-  padding: 0;
+  padding-top: @base-spacing-unit * 0.5;
+  padding-left: @base-spacing-unit * 4/3;
 
   /* -----------------------------------------------------------------------------
   Sidebar Menu Item
   ----------------------------------------------------------------------------- */
 
   .sidebar-menu-item {
 
+    .text-styling (
```

Comment:
![px](https://s3.amazonaws.com/f.cl.ly/items/0D40123W2w2A2N1u0z3r/Image%202016-05-03%20at%205.08.31%20PM.png?v=d822e97f)

sidetab fonts are smaller and not aligned to the icon on the left